README.md

```diff
@@ -6,42 +6,34 @@ with the Prometheus database. It consists of two parts:
      Forschungszentrum Jülich (short: FZJ). It does so by parsing the website
      providing the information.
   2. exporter (`main.py`): uses said script to receive, parse and provide
-     the data to the Prometheus database. Once started, it runs indefinitely until interrupt.
+     the data to the Prometheus database. Once started, it runs indefinitely
```

Comment:
Nit: "[...] data to the Prometheus database." implies a push model, which is isn't.

The exporter knows nothing about what is scraping it. Rather, it's providing data in a format that's specified for prometheus exporters.